src/olympia/files/tests/test_helpers.py

```diff
@@ -70,6 +70,16 @@ def test_recurse_contents(self):
         for name in nm:
             assert name in files
 
+    def test_recurse_contents_of_zip(self):
+        self.viewer.src = get_file('recurse.zip')
+        self.viewer.extract()
+        files = self.viewer.get_files()
+        nm = ['recurse/recurse.xpi/chrome/test-root.txt',
+              'recurse/somejar.jar/recurse/recurse.xpi/chrome/test.jar',
+              'recurse/somejar.jar/recurse/recurse.xpi/chrome/test.jar/test']
```

Comment:
One minor nitpick, I have absolutely no idea what `nm` stands for, can we call the object `file_list` or something more readable?